tool/tbot/main.go

```diff
@@ -158,6 +166,18 @@ func Run(args []string, stdout io.Writer) error {
 		"Arguments to `tsh proxy ...`; prefix with `-- ` to ensure flags are passed correctly.",
 	))
 
+	sshCmd := app.Command("ssh", "").Hidden()
+	sshCmd.Flag("destination-dir", "The destination directory with which to authenticate tsh").StringVar(&cf.DestinationDir)
+
+	sshProxyCmd := sshCmd.Command("proxy", "")
+	sshProxyCmd.Arg("[user@]host:port", "Remote host:port and the login to use").Required().StringVar(&cf.UserHostPort)
```

Comment:
We don't have a reason to merge the `%u@%h:%p` in a single string in `ssh_config` IMO.